qcfractal/interface/molecule.py

```diff
@@ -26,7 +26,22 @@ class Molecule:
 
     def __init__(self, mol_str, **kwargs):
         """
-        the Mongo QCDB molecule class which is capable of reading and writing many formats.
+        __init__(self, mol_str, name="", dtype=None, orient=False)
+
+        A molecule class which supports many different IO formats, provides unique molecule hashes, and
+        other molecular maniuplations.
+
+        Parameters
+        ----------
+        mol_str : str, dict
+            A string or dictionary representation of the molecule.
+        name : {"", str}, optional
+            The name of the molecule, defaults to "".
+        dtype : {None, "numpy", "json", "psi4"}, optional
+            Determines how the input is parsed.
+        orient : bool, optional
+            Orientate the molecule to a standard frame or not.
+
```

Comment:
Can we consolidate with molparse someday?

Reply:
Very much so!